tools/cooja/apps/mspsim/build.gradle

```diff
@@ -0,0 +1,55 @@
+apply plugin: "java"
+
+sourceSets {
+    main {
+        java {
+            srcDir 'src'
+        }
+    }
+}
+
+task buildMspsimJar(type: Exec) {
+    environment System.getenv()
+    workingDir '../../../mspsim/'
+    commandLine 'sh', '-c', 'ant jar'
+}
```

Comment:
I don't have windows for testing it. Do you have it?

2015-08-24 10:34 GMT+02:00 Moritz Strübe <notifications@github.com>:

> In tools/cooja/apps/mspsim/build.gradle
> <https://github.com/contiki-os/contiki/pull/771#discussion_r37729252>:
>
> > @@ -0,0 +1,55 @@
> > +apply plugin: "java"
> > +
> > +sourceSets {
> > +    main {
> > +        java {
> > +            srcDir 'src'
> > +        }
> > +    }
> > +}
> > +
> > +task buildMspsimJar(type: Exec) {
> > +    environment System.getenv()
> > +    workingDir '../../../mspsim/'
> > +    commandLine 'sh', '-c', 'ant jar'
> > +}
>
> Is this tested for windows?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/contiki-os/contiki/pull/771/files#r37729252>.
>